content/blog/2025-04-04-tmil-15/index.md

```diff
@@ -0,0 +1,212 @@
+- [xilem#899][]: Added documentation for lots of items and methods which were missing documentation.
+- [xilem#882][]: Adds the core of a virtual scrolling widget, to be used with medium or large scroll areas.
+
+<figure>
+<img style="height: auto" src="virtual-scroll.png" alt="A screenshot of a desktop window, showing a list of numbers around 999900. Every number divisible by 3 has been replaced with the word 'Fizz', 'Buzz' for 5, and 'FizzBuzz' for numbers divisible by both. The 'FizzBuzz' which would be item 999900 is twice as large as the other items." width="458" height="509">
+<figcaption>
+
+[xilem#882][] added a virtual scrolling widget. This is the 'Infinite FizzBuzz' example, which also shows how the virtual scrolling can support items with arbitrary heights.
+
+</figcaption>
+</figure>
+
+## Xilem
+
+Xilem is our flagship GUI project, inspired by SwiftUI, which uses Masonry for its widgets.
+It lets you build user interfaces declaratively by composing lightweight views together, and will diff them to provide minimal updates to a retained layer.
+
+- [xilem#887][]: Added a View for the already existing Split widget.
+- [xilem#899][]: Added some more documentation.
+
+## Vello
+
+Vello is our GPU vector renderer.
+It can draw large 2D scenes with high performance, using GPU compute shaders for most of the work.
+
+- [vello#841][]: Fixed incorrect COLR Emoi rendering, which especially impacted users on Windows.
+  We backported this to create [Vello 0.4.1](https://github.com/linebender/vello/releases/tag/v0.4.1).
+- [vello#817][], [vello#877][]: Updates some of our GPU encoding code to not assume that allocators will give over-aligned allocations.
+
+We have also renamed the #gpu channel on Zulip to [#vello](https://xi.zulipchat.com/#narrow/channel/197075-vello), which better reflects the purpose of the channel.
+Progress on the sparse strips renderer has also been continuing at pace:
+
+- [vello#827][]: Added some core API types.
+- [vello#828][]: Imported the implementation of sparse strip production, without rasterisation (Vello Common).
+- [vello#830][], [vello#832][]: Added on CPU rasterisation of sparse strips, with integration tests (Vello CPU).
+- [vello#834][]: Added a tool (Vello Toy) for visualising different stages of the new pipeline.
+- [vello#831][]: Added the fragment shader based GPU rasteriser for sparse strips (Vello Hybrid).
+- [vello#872][]: Enable running Vello Hybrid using WebGL2.
+- [vello#867][]: Benchmarking harness for Vello Common and Vello CPU.
+
+<figure>
+<img style="height: auto" src="paris-30k-webgl.png" alt="A browser window showing a partial map of the center of Paris, France." width="2256" height="752">
+<figcaption>
+
+Paris-30k rendered using Vello's new hybrid renderer in Firefox on Linux, a browser which does not yet support WebGPU.
+
+</figcaption>
+</figure>
+
+The current state of the sparse strip renderers is that they can reliably render scenes where all elements are a solid colour.
+Laurenz'
```

Comment:
I think this line should be removed, as I don't think Laurenz is ready for wider distribution of his thesis proposal yet.